README.md

```diff
@@ -353,6 +353,24 @@ julia> TimerOutputs.totallocated(to)
 7632
 ```
 
+## Exporting data
+
+The `TimerOutput` type implements the [Tables.jl](https://github.com/JuliaData/Tables.jl)
+interface. Thus, it works with various data-processing packages such as DataFrames.jl.
+For example, use `DataFrame(to)` (where `to isa TimerOutput`) to create a data frame 
+containing the timing and GC information.
+
+The output table has the following columns:
+
+* `name`: A tuple of strings. The i-th element is the label of the i-the level.
+* `ncalls`: The number of times `@timeit` is evaluated.
+* `time`: The time (nanoseconds) spent, excluding inner `@timeit` sections if any.
+* `tottime`: The time (nanoseconds) spent, including inner `@timeit` sections if any.
```

Comment:
Actually, these descriptions of `time` and `tottime` are incorrect? I thought above are the case, but then with v0.5.13 I get:

```
julia> to = TimerOutput();

julia> @timeit to "1" begin
           @timeit to "2" sleep(0.1)
           @timeit to "3" sleep(0.2)
           sleep(0.3)
       end;

julia> TimerOutputs.tottime(to)
603860055

julia> TimerOutputs.time(to["1"])
603860055

julia> TimerOutputs.tottime(to["1"])
302451267

julia> TimerOutputs.time(to["1"]["2"])
101178932

julia> TimerOutputs.time(to["1"]["3"])
201272335
```

It looks like `time` is already "total time" and `tottime` is the total time of the inner timers? Or maybe `tottime` is not supposed to be called with non-root timers? The README only mentions the usage with the root timer.

Reply:
I don't really recall what exactly it measures. Would need to read the source code.